ts/src/htx.ts

```diff
@@ -5701,7 +5701,7 @@ export default class htx extends Exchange {
         });
     }
 
-    async cancelOrders (ids, symbol: Str = undefined, params = {}) {
+    async cancelOrders (ids: string[], symbol: Str = undefined, params = {}): Promise<Order[]> {
```

Comment:
@ttodua you're updating the return type but we have to make sure the actual returned type matches, for instance here we just return the raw response 
![image](https://github.com/ccxt/ccxt/assets/43336371/821bd04d-a384-4ea9-a4fe-d18af3870b06)